job/job.go

```diff
@@ -113,7 +113,8 @@ func (u *Unit) IsGlobal() bool {
 	}
 	// Last value found wins
 	last := values[len(values)-1]
-	return strings.ToLower(last) == "true"
+	lastLower := strings.ToLower(last)
+	return lastLower == "true" || lastLower == "yes" || lastLower == "1" || lastLower == "on" || lastLower == "t"
```

Comment:
Break this logic out into a helper function and test it independently of Unit.IsGlobal